tests/check_coding_style.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# check_coding_style.sh
+# Copyright (c) 2019 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+# This is a checker for coding style issues specifically aimed at following the
+# coding style as published on:
+# http://endless-sky.github.io/styleguide/styleguide.xml
+#
+# This checker performs heuristic pattern matching, so there is a small risk that
+# constructs will be rejected that are actually allowed. The white-list should be
+# used to deal with such false positives and can also be used for allowed deviations.
+
+
+# Determine path of the current script
+HERE=$(cd `dirname $0` && pwd)
+SOURCES="${HERE}/../source"
+
+# Desired header, adapted for automatic parsing
+DESIRED_HEADER="/* FILENAME
+Copyright (c) 2xxx by Michael Zahniser
```

Comment:
It is not a strict requirement that all files are solely copyright MZ. Is that going to be flagged by this?

Reply:
The text now says:
`Copyright (c) 2xxx by NAME`
Do you think that is sufficiently clear?